### ISSUE TITLE
Update to 1.21.2+

### ISSUE THREAD

**Issue by Cecer** [CecerMCProjects/NoChangeTheGame#23]

**Comment by justino901** [comment]
do you think that by updating to 1.21.2 it will also make it work for 1.21.3 since they are similar from what i heard?

**Comment by lowercasebtw** [comment]
yes, but tbh this should be now updated to 1.21.4

**Comment by Cecer** [comment]
## Status Update
My available time has been reduced significantly over the last few months. Luckily, the primary feature of this mod, sword blocking, is now possible for servers to implement without any client mods as of recent versions.

I would still like to update the mod for the remaining features but I can't make any promises on when. The reduced demand also makes it hard to justify prioritising this over responsibilities projects.

### TLDR
- Sword blocking works in vanilla with the latest versions.
- Update is still planned but there is no ETA.
---

Related issue: #25

**Comment by lowercasebtw** [comment]
May I share something of interest?

**Comment by Fluboxer** [comment]
Fixing of sneaking is primary thing I use in this mod
Not getting lagged back when you shift + jump or not getting completely bugged out when you ender pearl into something that is not 100% perfect flat landing zone making is much more enjoyable
